.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
```

Comment:
What does this mean?

Reply:
as per GitHub workflow docs (https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release), this should trigger when we publish a release for the repo using GitHub

Reply:
this won't trigger if we create a tag and don't create a rlease

Reply:
previous releases for repo-tools can be found here: https://github.com/edx/repo-tools/releases

Reply:
It looks like there is support for releasing only on tags after test runs: https://github.com/marketplace/actions/pypi-publish#usage .  But honestly, this "deploy on release" approach seems simpler and less error prone; we're constantly hitting issues in Travis where we fail to deploy or attempt to deploy multiple times (causing the job to fail) because we got the conditions wrong in the CI file.  I guess the workflow would look like:

1. Merge the PR
2. Tag the commit
3. Make a GitHub Release for it (and then it will get automatically deployed to PyPI)

Another bonus for this is that we'd probably start putting halfway useful release notes in the GitHub releases area.  And only the 3rd step is new.  @nedbat , does this sound reasonable to you?

Reply:
(I had said, "Hmm, OK" yesterday, but GitHub's stupid "pending review" thing was hiding it...)

TBH, I didn't realize we were publishing this repo at all.  If Arch BOM likes the process, then I am happy.